css/templates/travel/reset.css

```diff
@@ -0,0 +1,48 @@
+/* http://meyerweb.com/eric/tools/css/reset/
+   v2.0 | 20110126
+   License: none (public domain)
+*/
```

Comment:
we should not be doing this. We already use normalize  for this

Reply:
Ok I'll look into removing this.

Reply:
Removed reset.css from this and the other two landing page sections.